lib/active_merchant/billing/gateways/worldpay_us.rb

```diff
@@ -151,7 +174,7 @@ def commit(action, post)
 
         raw = parse(ssl_post(live_url, post.to_query))
 
-        succeeded = success_from(raw['transresult'])
+        succeeded = success_from(raw['result'])
```

Comment:
The transresult field doesn't exist in responses to eChecks.  It is listed
in their docs and everything points to it being included in every response.


On Tue, Sep 15, 2015 at 4:48 PM, Ryan Daigle <notifications@github.com>
wrote:

> In lib/active_merchant/billing/gateways/worldpay_us.rb
> <https://github.com/activemerchant/active_merchant/pull/1874#discussion_r39562901>
> :
>
> > @@ -151,7 +174,7 @@ def commit(action, post)
> >
> >          raw = parse(ssl_post(live_url, post.to_query))
> >
> > -        succeeded = success_from(raw['transresult'])
> > +        succeeded = success_from(raw['result'])
>
> @mrezentes <https://github.com/mrezentes> this seems like a breaking
> change - could it have ever worked using transresult?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/activemerchant/active_merchant/pull/1874/files#r39562901>
> .
>



-- 
Miki Rezentes
Software Engineer